test/intl402/Locale/prototype/getWeekInfo/output-object-keys.js

```diff
@@ -53,12 +52,8 @@ let original = new Intl.Locale('en').getWeekInfo().weekend;
 let sorted = original.slice().sort();
 assert.compareArray(original, sorted);
 
-verifyProperty(result, 'minimalDays', {
-  writable: true,
-  enumerable: true,
-  configurable: true
-});
-assert(
-  isIntegerBetweenOneAndSeven(new Intl.Locale('en').getWeekInfo().minimalDays),
-  '`minimalDays` must be an integer between one and seven (inclusive)'
+assert.sameValue(
+  new Intl.Locale('en').getWeekInfo().minimalDays,
+  undefined,
+  '`minimalDays` is not defined.'
```

Comment:
This can probably just be deleted, as the `ownKeys` test above already ensures there isn't a `minimalDays` own property. (An implementation would still be technically, but uselessly, conformant if it added a `Object.prototype.minimalDays` property.)